lib/EpdFont/scripts/convert-builtin-fonts.sh

```
@@ -12,19 +40,15 @@ for size in ${NOTOSERIF_FONT_SIZES[@]}; do
   for style in ${READER_FONT_STYLES[@]}; do
     font_name="notoserif_${size}_$(echo $style | tr '[:upper:]' '[:lower:]')"
     font_path="../builtinFonts/source/NotoSerif/NotoSerif-${style}.ttf"
-    output_path="../builtinFonts/${font_name}.h"
-    python fontconvert.py $font_name $size $font_path --2bit --compress --pnum > $output_path
-    echo "Generated $output_path"
+    run_fontconvert "$font_name" "$size" "$font_path" --2bit --compress --pnum
   done
 done
 
 for size in ${NOTOSANS_FONT_SIZES[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)